R/utils.R

```diff
@@ -445,3 +471,26 @@ build_datanames_error_message <- function(label = NULL,
     }
   )
 }
+
+#' Pluralize a word depending on the size of the input
+#'
+#' @param x (`object`) to check length for plural.
+#' @param singular (`character`) singular form of the word.
+#' @param plural (optional `character`) plural form of the word. If not given an "s"
+#' is added to the singular form.
+#'
+#' @return A `character` that correctly represents the size of the `x` argument.
+#' @keywords internal
+pluralize <- function(x, singular, plural = NULL) {
```

Comment:
This new function really needed? Are there other cases were it will be used? If not, it will be simpler to just check on line 144 https://github.com/insightsengineering/teal/pull/1416/files#diff-60bef5af43a1bd541299ce9bceeb226afb23deaa5e637b145c85e05ffd552b0cR144 and chose one or the other sentence.

Reply:
I believe this will simplify the whole file, as we have a bunch of `if` and `ifelse` statements

I didn't commit yet to make sure everything works as expected.